client/app/deploy/services/loadBalancerService.js

```diff
@@ -0,0 +1,51 @@
+/* TODO: enable linting and fix resulting errors */
```

Comment:
Outstanding todo

Reply:
Yeah, I think we need a plan for this. I am really not feeling these es-lint rules.